build.js

```diff
@@ -0,0 +1,19 @@
+import fs from "fs";
+import HtmlBuilder from "./src/build_html/html_builder.js";
+import { JSDOM } from "jsdom";
+import path from "path";
+
+global.document = new JSDOM().window.document;
+
+const roadmapsFolder = "./src/configuration/roadmaps";
+const roadmapJsons = {};
+
+fs.readdirSync(roadmapsFolder).forEach(file => {
+  const json = JSON.parse(fs.readFileSync(path.join(roadmapsFolder, file), 'utf-8'));
+  roadmapJsons[file.split('.')[0]] = json;
```

Comment:
```suggestion
  const filename = path.join(roadmapsFolder, file);
  const fileData = fs.readFileSync(path.join(roadmapsFolder, file), 'utf-8');
  const json = JSON.parse(fileData);
  
  const fileWithoutExtension = file.split('.')[0];
  roadmapJsons[fileWithoutExtension] = json;
```

переменных вроде не жалко, а сразу понятнее.